octoprint_mrbeam/templates/wizard/wizard_acl.jinja2

```diff
@@ -4,14 +4,13 @@
     <strong>Please read the following, it is very important for the security of your Mr Beam 2!</strong>
 </p>
 <p>
-    Your Mr Beam 2 is configured with a Access Control, meaning you won't be able to do anything with the
+    Your Mr Beam 2 is configured with a Access Control, meaning you won\'t be able to do anything with the
```

Comment:
Alls strings in this file need to be translated. Looks like you forgot this file

Reply:
It is inside a block {% trans %} ... {% endtrans %}
The whole text in between extracted as one paragraph